docs/HACKING.md

```diff
@@ -74,7 +81,7 @@ The recommended developement environment uses a prebuilt OpenStack image.  This
 
 If your environment is different than the typical dev environment, you may be able to use the script as a reference or adapt it to your needs.  To execute the changes, run:
 
-    $ cd $PROJECT_HOME/goldstone-server
+    $ cd ~/devel/goldstone-server
     $ bin/stop_dev_env.sh
```

Comment:
My docker-vm was already powered off. So, stop_dev_env hung. If everyone has it not running at this point, this line should be deleted.